Userland/Utilities/xxd.cpp

```diff
@@ -261,9 +330,16 @@ ErrorOr<int> serenity_main(Main::Arguments args)
         }
     }
 
-    // TODO: colorize output
+    bool colorize_output = false;
     if (!colorize_output_option.is_null()) {
-        warnln("Colorizing output is not supported");
+        if (colorize_output_option == "always" || colorize_output_option == "auto") {
```

Comment:
I think `auto` should enable when writing to a tty and disable else. (Use something like `if (colorize_output_option == "always" || (colorize_output_option == "auto" && TRY(Core::System::isatty(STDOUT_FILENO))`)

Reply:
Thanks for introducing me to the function. I knew what to do for `auto`, just didn't know how (in Serenity.)